### ISSUE TITLE
Firewall security

### ISSUE THREAD

**Issue by mcadam** [xetys/hetzner-kube#36]
Hi, it would be nice to add some extra security and enable the firewall (ufw) in ubuntu. 

We will need some extra configuration to allow for access to Kubernetes api and other service you need to expose but easily we could add those in the provisioning script no?

**Comment by xetys** [comment]
yes of course. Do you want to make a PR for that?

**Comment by mcadam** [comment]
Yep let me try and create a PR if I can find some time in the next couple days.
